sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ScalaUDF.scala

```diff
@@ -37,21 +39,36 @@ case class ScalaUDF(
 
   override def toString: String = s"UDF(${children.mkString(",")})"
 
+  // Accessors used in genCode
```

Comment:
Why do we need manually created accessors?  All of the arguments to a case class should have public methods already created for them.